products/sle/main.pm

```diff
@@ -572,6 +573,9 @@ sub load_reboot_tests() {
         }
         if (get_var('ENCRYPT')) {
             loadtest "installation/boot_encrypt";
+            if (check_var('BACKEND', 'svirt')) {
+                loadtest "installation/reconnect_s390";
+            }
```

Comment:
I also added reconnect behind boot_encrypt to make it pass.

[failed](http://10.100.12.155/tests/5082)
[passed](http://10.100.12.155/tests/5086)